Dockerfile.template

```diff
@@ -30,14 +30,12 @@ LABEL summary=$SUMMARY \
       name="{{ spec.org }}/postgresql-{{ spec.short }}-{{ spec.prod }}" \
       com.redhat.component="{{ spec.redhat_component }}" \
 {%- if spec.version in spec.rhscl3_introduced %}
-      version="1"
+      version="1" \
 {%- else %}
       version="{{ spec.version }}" \
-      release="1"
-{%- endif %}
-{%- if spec.maintainer %} \
-      maintainer="{{ spec.maintainer }}"
 {%- endif %}
+      usage="docker run -d --name postgresql_database -e POSTGRESQL_USER=user -e POSTGRESQL_PASSWORD=pass -e POSTGRESQL_DATABASE=db -p 5432:5432 {{ spec.org }}/postgresql-{{ spec.short }}-{{ spec.prod }}" \
+      maintainer="SoftwareCollections.org <sclorg@redhat.com>"
```

Comment:
While we are touching this, shouldn't maintainer= be set only for Fedora dockerfiles?

Reply:
@hhorak what do you think? Maintainer has been added to every Dockerfile in this commit: https://github.com/sclorg/postgresql-container/commit/b66f12fdc924c49b3f9b2f657546365f15d9bf93

Reply:
Last time we had this conversation we got to the conclusion that it is ok to leave this in if it simplifies things. But this should really get documented somewhere in our guidelines since are coming back to it...

Reply:
Given we set the maintainer label in s2i-core, it should be inherited to other containers, so omitting it is probably fine, but I don't see any problem with re-defining it, so if it makes the template simpler, I'm fine with that.